src/mainwindow.cpp

```diff
@@ -539,6 +539,8 @@ MainWindow::MainWindow(Settings &settings
   processUpdates();
 
   ui->statusBar->updateNormalMessage(m_OrganizerCore);
+
+  m_OrganizerCore.userInterfaceInitialized(this);
```

Comment:
Not sure about this. There's still more stuff to do in `showEvent()` and I don't like python code potentially accessing the main window object _before_ its constructor has finished.

Reply:
I agree, I can trigger right after `show()` in `main.cpp` or at the end of `showEvent()`.

Reply:
I've moved the call to the end of `showEvent`, see f8bb349